.github/workflows/repo-health-job.yml

```diff
@@ -137,10 +166,36 @@ jobs:
         if: ${{ inputs.DASHBOARD_NAME == 'repo_health' }}
         run: |
           bash edx-repo-health/scripts/repo-health-artifact.sh
-        
+
       - name: Upload sqlite db as artifact
         if: ${{ inputs.DASHBOARD_NAME == 'repo_health' }}
         uses: actions/upload-artifact@v4
         with:
           name: sqlite db artifact
           path: dashboards/dashboard.sqlite3
+
+      - name: Send email on failure
+        uses: dawidd6/action-send-mail@v3
+        if: ${{ failure() && inputs.ENABLE_EMAIL_ALERTS }}
+        with:
+          server_address: email-smtp.us-east-1.amazonaws.com
+          server_port: 465
+          username: ${{secrets.EDX_SMTP_USERNAME}}
+          password: ${{secrets.EDX_SMTP_PASSWORD}}
+          subject: Dependencies health job workflow failed in ${{github.repository}}
+          to: ${{ secrets.EMAIL_ADDRESS_FOR_ALERTS }}
```

Comment:
This can be stored in inputs instead

Reply:
Just following the convention here. Other email addresses are also being passed in secrets.